tests/integration/test_eoc_journal.py

```diff
@@ -196,6 +262,44 @@ def test_pb_answers_not_listed_when_none_available_in_student_view(self):
             self.get_element_for_selected_pb_answers
         )
 
+    def test_progress_metrics_listed_in_student_view(self):
+        self.set_standard_scenario()
+        self.go_to_view('student_view')
+        self.fix_js_environment()
+        
+        element = self.get_element_for_course_progress()
+        user_score = element.find_element_by_css_selector('span[data-progress-name="user"]')
+        self.assertEqual(float(user_score.text), default_progress_metrics['completions'])
+
+        cohort_score = element.find_element_by_css_selector('span[data-progress-name="cohort"]')
+        self.assertEqual(float(cohort_score.text), default_progress_metrics['course_avg'])
+
+    def test_engagement_metrics_listed_in_student_view(self):
+        self.set_standard_scenario()
+        self.go_to_view('student_view')
+        self.fix_js_environment()
+        
+        element = self.get_element_for_course_engagement()
+
+        score = element.find_element_by_css_selector('span[data-engagement-name="user"]')
+        expected = default_engagement_metrics['user_score']
+        self.assertEqual(float(score.text), expected)
+
+        score = element.find_element_by_css_selector('span[data-engagement-name="cohort"]')
+        expected = default_engagement_metrics['cohort_score']
+        self.assertEqual(float(score.text), expected)
+
+        d = default_engagement_metrics.copy()
+        d.pop('user_score')
+        d.pop('cohort_score')
```

Comment:
Nit: instead of popping the keys, we could move everything into the `for` loop and do:

```python
for key, value in default_engagement_metrics.iteritems():
    if key == 'user_score':
        tag = 'span[data-engagement-name="user"]'
    elif key == 'cohort_score':
        tag = 'span[data-engagement-name="cohort"]'
    else:
        tag = 'td[data-point-name="{}"]'.format(key)